.github/workflows/checks.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: poetry install --no-interaction
       - name: Run tests
         working-directory: ./cli
-        run: poetry run python -m pytest deduplifhirLib/tests.py
+        run: poetry run python -m pytest deduplifhirLib/tests/tests.py
```

Comment:
Is this because it's trying to run non-Python files? We should be able to address that in the configuration to make this more flexible going forward

Reply:
I fixed the invocation so that it runs on the directory